src/pytest_markdown_docs/plugin.py

```diff
@@ -123,15 +124,20 @@ def get_prefixed_strings(
 
 
 def extract_fence_tests(
+    markdown_it_parsers: typing.List["MarkdownIt"],
```

Comment:
I might be missing something here, but why is this a list? (seems like we are throwing away anything except the first element anyway)

Reply:
Ah I see, yeah this makes sense! Thanks for the explanation and I like the firstresult and default implementation